### DISCUSSION TITLE
DataTable row removal

### DISCUSSION THREAD

**Discussion by Marti2203** [Textualize/textual#2116] [Q&A] [answered]
Currently the DataTable does not support row removals. Is this intended?

**Comment by TomJGooding** [comment, marked as answer]
Apparently this feature is planned - see [this discussion](https://github.com/Textualize/textual/discussions/1856).

**Comment by Marti2203** [comment]
Hi, thanks for the response! I see... no response from the main devs though :\

**Comment by davep** [comment]
> I see... no response from the main devs though

Your question was asked at 03:04 GMT. All devs are in the GMT timezone. Reading your question was one of the first things I did after getting to the office, about 5 hours later. I hope everyone can agree that it's tricky for us to answer questions in our sleep, and that 5 hours isn't a terrible turnaround for an answer to a question asked of an Open Source project.

Thanks to @TomJGooding for giving the answer.

**Reply by Marti2203** [reply]
Nono, I meant about the previous discussion as it is being planned. Sorry, did not want to sound like an asshole :(

**Reply by davep** [reply]
Last I checked I was one of the main devs and I did answer in that discussion too. ;-)

**Reply by Marti2203** [reply]
True... My bad...

**Comment by willmcgugan** [comment]
I've added a ticket for this so you can track progress... https://github.com/Textualize/textual/issues/2118

**Reply by Marti2203** [reply]
Thank you, I would be happy to work on it personally. It is a feature for a project where we are using Textualize and I think that the codebase is easy enough to understand and would be happy to collaborate with you and @davep :)

**Reply by willmcgugan** [reply]
Happy to accept a PR if you want to tackle it!
